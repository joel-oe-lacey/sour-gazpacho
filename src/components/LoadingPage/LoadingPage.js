import React from 'react';
import './LoadingPage.scss';

const LoadingPage = () => {
    return (
        <section className="loading">Loading Movies</section>
    )
}

export default LoadingPage;