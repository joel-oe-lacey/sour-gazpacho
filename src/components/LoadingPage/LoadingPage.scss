.loading {
  height: 92%;
  width: 100%;
}