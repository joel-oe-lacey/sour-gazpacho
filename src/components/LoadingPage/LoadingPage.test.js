import React from 'react';
import LoadingPage from './LoadingPage';
import { shallow } from 'enzyme'

describe('LoadingPage', () => {


  it('should match a snapshot as expected', () => {
    const wrapper = shallow(<LoadingPage />)

    expect(wrapper).toMatchSnapshot()
  })
})
