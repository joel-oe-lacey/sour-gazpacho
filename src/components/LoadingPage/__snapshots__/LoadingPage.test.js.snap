// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should match a snapshot as expected 1`] = `
<section
  className="loading"
>
  <h1
    className="loading-header"
  >
    Loading Movies . . .
  </h1>
  <div
    className="lds-spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</section>
`;
